src/View.php

```diff
@@ -266,7 +266,7 @@ protected function findTemplateFile(string $view, ?ViewContextInterface $context
         } elseif ($context instanceof ViewContextInterface) {
             // path provided by context
             $file = $context->getViewPath() . '/' . $view;
-        } elseif (($currentViewFile = $this->getRequestedViewFile()) !== false) {
+        } elseif (is_string($currentViewFile = $this->getRequestedViewFile())) {
```

Comment:
What else could it be?

Reply:
It could be false when rendering from View directly
You can reproduce this behavior with the given method of the controller
`
public function index(WebView $view, DataResponseFactoryInterface $responseFactory): ResponseInterface
    {
        return $responseFactory->createResponse($view->render('index'));
    }`